packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -9570,37 +9554,23 @@ Generated by [AVA](https://ava.li).
       1. onPreBuild command from ./plugin                           ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-    Netlify configuration property "Symbol(test)" value changed to true.␊
+    ␊
+    (./plugin onPreBuild completed in 1ms)␊
     ␊
     ────────────────────────────────────────────────────────────────␊
-      Plugin "./plugin" internal error                              ␊
+      2. build.command from netlify.toml                            ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-      Error message␊
-      Error: "netlifyConfig.Symbol(test)" is read-only.␊
+    $ node --version␊
```

Comment:
Symbols are now handled slightly differently, which explains the test snapshot change.
They are now just ignored.